test/unit/behavior.spec.js

```diff
@@ -682,12 +667,6 @@ describe('Behavior', function() {
 
       expect(listenToChangeStub).not.to.have.been.calledOnce;
     });
-
-    it('should still be bound to "on" on destroy', function() {
```

Comment:
Not true anymore?

If so maybe test that is not bound on destro

Reply:
yep not true anymore.  `view.destroy().triggerMethod('foo');` will call `onFoo` on the view, but no longer on the view's behaviors.  This test was essentially duplicating the test to make sure `onDestroy` was called after the `destroy`  But testing that events listened to on the behavior itself are cleaned up on destroy is already being tested here: https://github.com/marionettejs/backbone.marionette/blob/next/test/unit/behavior.spec.js#L861